source/server/LocalServer.ts

```diff
@@ -65,47 +63,15 @@ module Print.Server {
 					break;
 				case "GET":
 					var urlPathList: string[] = url.pathname.split("/");
-					var cookieValue: string = LocalServer.getCookieValue(request.headers.cookie, "authorized");
-					var accessToken: string = this.findAccessToken(LocalServer.getCookieValue(request.headers.cookie, "authorized"));
-					if (url.query.error) {
-						console.log("Github ERROR: [" + url.query.error + "] Description: [" + url.query.error_description + "] Uri: [" + url.query.error_uri + "]");
-						LocalServer.sendResponse(response, 400, "Github error. See error message in server log");
-					}
-					else if (url.query.authorized == "no") {
-						this.fetchAccessToken(response, url);
-					}
-					else if (!accessToken) {
-						if (url.pathname == "/")
-							var redirectUrl = this.baseUrl + "/" + this.clientRoot
-						else
-							var redirectUrl = this.baseUrl + url.pathname
-						response.writeHead(301, { Location: "https://github.com/login/oauth/authorize?scope=" + this.githubScopes + "&client_id=" + this.serverConfig.getClientId() + "&redirect_uri=" + redirectUrl + "?authorized=no" });
+					if (url.pathname == "/") {
```

Comment:
This is to support the client only entering the webserver address. The github auth fixed this before